main.go

```diff
@@ -12,10 +12,10 @@ import (
 )
 
 var (
-	//go:embed dist
+	// go:embed dist
```

Comment:
This breaks the system. evcc now panics on startup as the embed does not happen and internal Filesystems are empty.

Reply:
das hat bei mir irgendwie vscode mit eingebracht.. habe es ohne nachdenken gepushed.. sorry xD

Reply:
Hab ich übersehen- aber VScode macht das egtl. nicht

Reply:
Konnte es bei mir nun nachvollziehen.. in vscode wurde mir da ein Fehler angezeigt `pattern dist: no matching files found` ..hatte es dann wohl manuell geändert. Komischerweise sind bei mir local keine negativen Seiteneffekte aufgetreten,

Reply:
Ja, bei go eun nutzen wir das live Filesystem statt des embeddeten.